composer.json

```diff
@@ -6,24 +6,26 @@
 		"jahuty"
 	],
 	"license": "MIT",
-	"authors": [
-		{
-			"name": "Jack Clayton",
-			"email": "jack@jahuty.com"
-		}
-	],
+  "homepage": "https://www.jahuty.com/",
+  "authors": [
+    {
+      "name": "Jahuty and contributors",
+      "homepage": "https://github.com/jahuty/jahuty-php/contributors"
+    }
+  ],
 	"require": {
 		"php": "^7.3",
-        "guzzlehttp/guzzle": "^6.3"
+    "guzzlehttp/guzzle": "^6.3"
```

Comment:
We might want to allow multiple versions of guzzle for wider usage. We can run different builds with `--prefer-lowest` and `--prefer-stable` to test all deps

Reply:
Do you mean relaxing the requirements to a major version number constraint only (i.e., `^6`), or do you mean allowing a number of major versions (i.e., something like `^5 || ^6` (my composer version constraint-foo is a little rusty haha)?

Reply:
allowing a number of major versions!
```
^5.3.4 || ^6.5.5 || ^7.2.0
```

Reply:
determining the differences in the usages of the library could be a little annoying (a lot of class_exists/method_exists calls) so the benefits need to be weighed here

Reply:
Sorry my friend! I didn't mean to merge these changes and ignore your suggestion here. At the time, I intended to move your idea into a separate issue... I just forgot to do that until now. See #16. Thanks!